scripts/trivia.js

```diff
@@ -23,6 +23,8 @@ var triviabot = new Bot("Metagross");
 var triviaCategories = ['Anagram: Pokémon', 'Anime/Manga', 'Animals', 'Art', 'Comics', 'Food/Drink', 'Games', 'Geography', 'History', 'Internet', 'Language', 'Literature', 'Math', 'Mental Math', 'Miscellaneous', 'Movies', 'Music', 'Mythology', 'Pokémon', 'Pokémon Online', 'Politics', 'Religion', 'Science', 'Social Science', 'Society', 'Space', 'Sports', 'Technology', 'Television', 'Video Games'];
 var specialCategories = ['Mental Math'];
 var lastCatGame = 0;
+var lastEventType = 'None.';
+var lastEventTime = sys.time();
```

Comment:
(20:37:39) ±Dratini: Type: 'function'
(20:37:39) ±Dratini: Got from eval: 'function time() { [native code] }'

I'm not sure that's what you are looking for there. You could leave it undefined or use something like ```new Date().getTime()```